pulp_rpm/tests/functional/api/test_publish.py

```diff
@@ -72,6 +72,45 @@ def test_publish_any_repo_version(
             }
             rpm_publication_api.create(body)
 
+    @pytest.mark.parametrize("compression_type", ("gz", "zstd"))
+    @pytest.mark.parallel
+    def test_publish_with_compression_types(
+        self,
+        compression_type,
+        rpm_unsigned_repo_immediate,
+        rpm_publication_api,
+        gen_object_with_cleanup,
+        rpm_distribution_api,
+        monitor_task,
+    ):
+        """Sync and publish an RPM repository w/ zstd compression and verify it exists."""
+        # 1. Publish and distribute
+        publish_data = RpmRpmPublication(
+            repository=rpm_unsigned_repo_immediate.pulp_href, compression_type="zstd"
```

Comment:
The `compression_type` value here isn't actually parameterized, it's only using zstd

Reply:
Oh, my bad